### ISSUE TITLE
`/say` does not display on Discord if ran by the Server

### ISSUE THREAD

**Issue by DorkOrc** [Reimnop/Discord4Fabric#28]
Messages sent through the console, through a function ran with no context, or as a non-player entity are not sent to Discord.

Messages sent through the console or ran by a contextless function should be sent to Discord with the author "Server" (this is most useful as it allows servers to send messages to Discord from in-game commands)
Messages sent through a non-player entity should either have that entity's name or a generic name

**Comment by Reimnop** [comment]
Right, it should not display any message if ran through the say command.

**Comment by Reimnop** [comment]
Fair point, I'll think about it
